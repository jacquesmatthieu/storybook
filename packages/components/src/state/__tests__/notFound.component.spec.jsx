import React from 'react';
import { shallow } from 'enzyme';

import NoFound from '../notFound.component';

describe('NoFound', () => {
  let props;

  beforeEach(() => {
    props = {
      steps: ['foo', 'bar'],
    };
  });

  const getWrapper = () => shallow(<NoFound {...props} />);

  it('should render properly', () => {
    // When
    const wrapper = getWrapper();

    // Then
    expect(wrapper).toMatchSnapshot();
  });
});
