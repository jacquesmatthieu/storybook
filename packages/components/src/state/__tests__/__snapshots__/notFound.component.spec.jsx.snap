// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoFound should render properly 1`] = `
<notFoundcomponent__ErrorWrapper
  data-test-id="notFoundError"
>
  <typographycomponent__Typography
    variant="headline"
  >
    error.title
  </typographycomponent__Typography>
  <br />
  <typographycomponent__Typography
    variant="body"
  >
    error.notFoundError
  </typographycomponent__Typography>
</notFoundcomponent__ErrorWrapper>
`;
